R/boundsPredprob.R

```diff
@@ -1,74 +1,86 @@
-#' Decision cutpoints for boundary (based on predictive probability)
+#' Decision cutpoints for boundary (based on predictive probability) for Decision 1 rule.
 #'
 #' This function is used to identify the efficacy boundary and futility
-#' boundary based on predictive probabilities, i.e.:
+#' boundary based on the following rules:
 #' Efficacy boundary: find minimum x (xU) where
-#' Pr(Pr(P > p | x, Y) >= tT | x) > phiU,
+#' Pr(Pr(P > p0 | x, Y, a, b) >= tT | x) >= phiU,
 #' Futility boundary: find maximum x (xL) where
-#' Pr(Pr(P > p | x, Y) >= tT | x) < phiL
+#' Pr(Pr(P > p0 | x, Y, a, b) >= tT | x) =< phiL
```

Comment:
do we always include `a` and `b` on the condition side of these probabilities? It is not really precise because it might also be a mixture of beta distributions as a prior (i.e. not just a and b as prior parameters), so I am not sure this is needed/useful